### DISCUSSION TITLE
Getting iostat=2 error when running anything inside exe/ directory.

### DISCUSSION THREAD

**Discussion by ppokhrel1** [NOAA-EMC/WW3#523] [Q&A]
Hi, 

I installed the Wavewatch in my computer and got to the compilation part. 

However, when I run anything inside the exe/ folder, I get error in opening file error. It looks something like this.

```
 *** WAVEWATCH III ERROR IN W3GRID : 
     ERROR IN OPENING INPUT FILE
     IOSTAT =    2

```
I was under impression that the file would not have compiled if I didn't have proper fortran. Could anyone help me on whether I am missing something during installation or it is a bug on WaveWatch ?

**Comment by aliabdolali** [comment]
It seems you do not have your input (*.inp/*.nml) in your work directory. Besides compiling the code, you need to have a setup including inputs files.

**Reply by alexusperplexus** [reply]
Hi, I have made sure I have those input files but it still gives me the same error, I am using WW3 through a docker image, could it be an error from the docker image? Thank you
